modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/layouts/02_my_evp/01_organization-form/page-definition.json

```diff
@@ -0,0 +1,573 @@
+{
```

Comment:
Hi Bruno, could you add a button for accessing the form in MyEVP. The way it is now, I have no way to open it, just by editing the page.